src/Sylius/Bundle/UserBundle/Form/Type/CustomerGuestType.php

```diff
@@ -50,8 +48,7 @@ public function buildForm(FormBuilderInterface $builder, array $options = array(
             ->add('email', 'email', array(
                 'label' => 'sylius.form.customer.email',
             ))
-            ->addEventSubscriber(new CustomerRegistrationFormListener($this->customerRepository))
-            ->setDataLocked(false)
```

Comment:
If we want the line https://github.com/Sylius/Sylius/pull/3372/files#diff-049be7c4c29104ef1af18573471fd854R68 to work when we swap the customer with the existing one, we need this line.

There is no scenario for that, but should be. Can you write a scenario for placing an order as a guest for already existing customer?